templates/kstreams/static/dev/code/docker-compose-up.sh

```diff
@@ -1 +1 @@
-docker-compose up
+docker-compose up -d
```

Comment:
piggy-backing a change for all generated tutorials.  Original suggestion from @rspurgeon.